src/Utility/Hash.php

```diff
@@ -296,7 +296,7 @@ protected static function _matches($data, string $selector): bool
      *
      * @param array $data The data to insert into.
      * @param string $path The path to insert at.
-     * @param array|null $values The values to insert.
+     * @param mixed $values The values to insert.
```

Comment:
Can we keep this more specific one?

Reply:
No since you can insert any type of value to particular key.

Reply:
Isnt this iterating inside, so it must be an array of things - saying mixed means you could also work on random objects like Response class, I doubt that is possible or intended.
We usually annotate the intended target input, not all remotely possible ones you could pass without crashing it IMO.

Reply:
Fixing incorrect type was more valuable than listing the possible one. You are welcome to check the code / test case and list the possible types instead.